_posts/en/2018-08-30-dashboard-announcement.md

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Bitcoin Optech Dashboard'
+permalink: /en/dashboard-announcement/
+name: 2018-08-30-dashboard-announcement
+type: posts
+layout: post
+lang: en
+version: 1
+---
+
+{:.post-meta}
+*by [Marcin Jachymiak](https://github.com/marcinja)<br>Intern at Bitcoin Optech*
+
+This summer I was an intern for [Bitcoin Optech](https://bitcoinops.org), working on a Bitcoin metrics dashboard. In this post I'll be describing the purpose of the dashboard and how it was implemented. 
+
+The goal of the Optech dashboard is to show a variety of metrics of how effectively blockspace is being used. Important metrics are those that show activity like: 
+ - [batching](https://en.bitcoin.it/wiki/Techniques_to_reduce_transaction_fees#Payment_batching): combining the outputs of what would otherwise be many different transactions into a single transaction
+ - [consolidations](https://en.bitcoin.it/wiki/Techniques_to_reduce_transaction_fees#Consolidation): combining many UTXOs in a single UTXO during low fee period, to avoid paying a higher fee in the future
+ - dust creation: creation of UTXOs that cost more to spend than they have value, at different fee-rates
```

Comment:
Since you've included links for the previous two, how about https://bitcoin.stackexchange.com/questions/10986/what-is-meant-by-bitcoin-dust for dust and https://bitcoincore.org/en/2016/01/26/segwit-benefits/ or https://en.bitcoin.it/wiki/Techniques_to_reduce_transaction_fees#P2SH-wrapped_segwit for segwit.